### PR TITLE
Update suma form with new entries

### PR DESCRIPTION
**Pull request by Simranpal**
Adding some missing entries for the suma form for HANA. 
I need to validate with Suse Manager yet.

### CODE REVIEW

File: form.yml [hunk 1]
```diff
@@ -6,6 +6,51 @@ hana:
     $type: boolean
     $default: true
     $help: Install all required packages from currently existing repositories
+    $optional: true
+  saptune_solution:
```
Comment (arbulu89) [inline form.yml:10]: This should have `HANA` as default, right?

**Review by arbulu89 — COMMENTED** [review]
@Simranpal Is this PR finished and tested?

**Comment by arbulu89** [comment]
@Simranpal How is the state of this PR? It looks finished

**Comment by MalloZup** [comment]
@Simranpal @arbulu89  I think we should work to get this merged asap/this sprint for suma submission of our formulas.

JFYI :tea:

**Comment by arbulu89** [comment]
@MalloZup I agree, but It has been stalled for months. I want to create a new release for https://github.com/SUSE/ha-sap-terraform-deployments and I won't wait more.
We might do the new maintenance submissions today, and this will need to wait, and will come in a new submission

File: form.yml (hunk 1 above)
Comment (Simranpal) [reply form.yml:10]: Thanks @arbulu89 yes I will update this.

**Comment by Simranpal** [comment]
> @Simranpal How is the state of this PR? It looks finished

Yes @arbulu89 I will test this along with ha formula one, and update if any new changes.

**Comment by arbulu89** [comment]
This PR looks strange. It's bringing commits that are already in master

**Comment by Simranpal** [comment]
> This PR looks strange. It's bringing commits that are already in master

I was testing something with git, and somehow the branches got diverged, perhaps i can create a new pr

**Pull request closed without merging**